frontend/src/styles/components/commentform.css

```diff
@@ -19,4 +19,5 @@
 .length {
```

Comment:
is this space preventing from applying the styling you have here? please check and remove if so!

Reply:
Doesn't seem to be a space

Reply:
Yes, it is not a space :)